storage-node/src/cache/mod.rs

```diff
@@ -0,0 +1,18 @@
+use self::lru::LruCache;
+
+pub mod lru;
+
+pub type ParpulseCacheKey = String;
+pub type ParpulseCacheValue = String;
```

Comment:
I guess the value should be a tuple of a string and a usize element, instead of purely a string.